test/functional/api/buffer_spec.lua

```diff
@@ -38,7 +38,7 @@ describe('api/buf', function()
 
     it('line_count has defined behaviour for unloaded buffers', function()
       -- we'll need to know our bufnr for when it gets unloaded
-      local bufnr = buffer('get_number', 0)
+      local bufnr = curbuf('get_number')
```

Comment:
I meant just `curbuf()` with no args.

Reply:
I need the buffer number to use in `command('bunload! '..bufnr)`, but I can't figure out how to get the buffer number from the handle returned by `curbuf()` with no args.

Reply:
The handle returned by `curbuf()` is the buffer number.